.github/workflows/release.yml

```diff
@@ -178,7 +186,7 @@ jobs:
           git reset --hard
           # pull latest version to ensure the required commit with GDAL 3.8 is available
           git pull
-          git checkout 4f4a1821b2e8c7a2863e4df65a4d514f84144049
+          git checkout 73794ce5f63fd138fab999a22959ca7c6305d93c
 
       - name: Install GDAL
         env:
```

Comment:
There is a `--overlay-triplets=./ci/custom-triplets` a few lines below here that can then be removed as well

Reply:
Don't we still need to use the custom triplets?